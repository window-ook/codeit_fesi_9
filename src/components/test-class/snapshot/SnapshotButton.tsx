export default function SnapshotButton({
    children,
}: {
    children: React.ReactNode;
}) {
    return <button>{children}</button>;
}