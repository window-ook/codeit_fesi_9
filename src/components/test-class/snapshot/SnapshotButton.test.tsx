import { render } from "@testing-library/react";
import SnapshotButton from "@/components/test-class/snapshot/SnapshotButton";

test("Button 컴포넌트가 올바르게 렌더링됩니다", () => {
    const { container } = render(<SnapshotButton>클릭하세요</SnapshotButton>);
    expect(container).toMatchSnapshot();
});
