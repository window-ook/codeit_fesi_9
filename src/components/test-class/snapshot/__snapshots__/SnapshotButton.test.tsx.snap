// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button 컴포넌트가 올바르게 렌더링됩니다 1`] = `
<div>
  <button>
    클릭하세요
  </button>
</div>
`;
